### ISSUE TITLE
Issues with spanish import process

### ISSUE THREAD

**Issue by felixilef2** [bardsoftware/ganttproject#1363]
I downloaded the latest version and it is automatically working in spanish. I have exported an example file to csv in order to get the file format. Then I modified it and tried to import again, but some of the columns are not working as it does not seem to understand the accents marks (á, é, etc) that it (itself) wrote.
How can it get to work in english to avoid this issue?

**Comment by dbarashev** [comment]
How do you know that it is accents which breaks the import? I just've tried to export and import House Building Sample and although I see that resource roles are lost (all resources got undefined role), I also see that columns with accents in names are perfectly kept. Moreover, resource roles are lost with English translation as well.

Can you give more details on what columns are "not working" (and what does "not working" mean) ?

**Comment by felixilef2** [comment]
Hello Dmitry.
Thankyou very much for your prompt reply.
I was trying to bring data from two excel sheets, one with tasks and the other with resources. I was generating a csv file wit VBA, and the column names were copied from the csv file I had exported from GP. Then I inserted the rows from the excel sheets.As I was having problems with this and I saw that the columns containing words with accents went to "custom columns" with "?" instead of "ó", for example, I took for granted that there was some issue with this. Now I realize that the big problems I had were some mistakes in my original excel sheets (resources names in tasks sheet not exactly the same as the ones in the resources sheet). Once I have solved this, I am having problems with the dates, but I will investigate it in the forums before asking again.
Anyway I suggest that you use capital letters for column heads, as with these in spanish it is not mandatory to use accents.
Thankyou again, and sorry for any inconvenience.
You are doing a really good job. 

    El Sábado 11 de marzo de 2017 3:08, Dmitry Barashev <notifications@github.com> escribió:
 

 How do you know that it is accents which breaks the import? I just've tried to export and import House Building Sample and although I see that resource roles are lost (all resources got undefined role), I also see that columns with accents in names are perfectly kept. Moreover, resource roles are lost with English translation as well.Can you give more details on what columns are "not working" (and what does "not working" mean) ?—

**Comment by dbarashev** [comment]
[Issue1363.zip](https://github.com/bardsoftware/ganttproject/files/836987/Issue1363.zip)
I just've tried to add a custom column with accented character, added a value with accented character, exported to CSV and imported back. It works as expected, everything imports fine. The original file and exported CSV are attached.

**Comment by felixilef2** [comment]
Hello Dmitry.
I just imported your file and it works as you said. I copy-pasted the first line in my csv file and it still didn't work. Then I did it the other way and copy-pasted from my csv into your file and voilá, it did work. It seems to me that it is because I am writing from excel encoding with ASCI and you are using UTF-8. I will try to change it in excel.
Thank you very much. 

    El Lunes 13 de marzo de 2017 0:29, Dmitry Barashev <notifications@github.com> escribió:
 

 Issue1363.zip
I just've tried to add a custom column with accented character, added a value with accented character, exported to CSV and imported back. It works as expected, everything imports fine. The original file and exported CSV are attached.—
